extensions/wikia/Recirculation/js/recirculation.js

```diff
@@ -20,6 +20,8 @@ require([
 	'use strict';
 
 	var $mixedContentFooter = $('#mixed-content-footer'),
+		$mixedContentFooterContent = $('.mcf-content'),
+	    wikiArticlesFetched = false,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'wikiArticlesFetched' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)